src/ReactLeafletMap/layers/TiledVectorLayer/Tile.js

```diff
@@ -0,0 +1,103 @@
+import React from 'react';
+import _ from 'lodash';
+import PropTypes from 'prop-types';
+import {withLeaflet} from "react-leaflet";
+import memoize from "memoize-one";
+
+import IndexedVectorLayer from "../IndexedVectorLayer";
+import VectorLayer from "../VectorLayer";
+
+/**
+ * @param featureKeysGroupedByTileKey {Array} A collection of feature keys by tile key
+ * @param tileKey {String} unique tile identifier
+ * @param features {Array} List of current tile's features
+ * @param fidColumnName {String}
+ * @return {[]|null} List of feature keys to omit
+ */
+function getFeatureKeysToOmit(featureKeysGroupedByTileKey, tileKey, features, fidColumnName){
+	// Find the order of current tile among others
+	const indexOfCurrentTile = _.findIndex(featureKeysGroupedByTileKey, (tile) => {return tile.tileKey === tileKey});
+
+	let i = 0;
+	if (indexOfCurrentTile > 0) {
+		let featureKeysToOmit = [];
+		let renderedFeatureKeys = new Set();
+
+		// Iterate over sibling tiles that should be rendered earlier
+		while (i < indexOfCurrentTile) {
+			const tile = featureKeysGroupedByTileKey[i];
+			_.forEach(tile.featureKeys, featureKey => {
```

Comment:
I'm not sure, if this is faster. I always add just one key, so I used the add method

Reply:
But then you do not need to use forEach.

Reply:
I'm not sure that it will be a significant performance improvement. See the comments here: https://stackoverflow.com/questions/32000865/simplest-way-to-merge-es6-maps-sets/50296208

Reply:
agree